test/OpenTelemetry.Instrumentation.Http.Tests/HttpWebRequestActivitySourceTests.netfx.cs

```diff
@@ -544,8 +544,8 @@ public async Task TestRequestWithException(string method)
             Assert.True(eventRecords.Records.TryDequeue(out KeyValuePair<string, Activity> exceptionEvent));
             Assert.Equal("Stop", exceptionEvent.Key);
 
-            Assert.NotNull(activity.GetTagValue(SpanAttributeConstants.StatusCodeKey));
-            Assert.NotNull(activity.GetTagValue(SpanAttributeConstants.StatusDescriptionKey));
+            Assert.True(exceptionEvent.Value.Status != ActivityStatusCode.Unset);
```

Comment:
Assert on activity status and statusdescription

Reply:
Vishwesh and I chatted offline and we confirmed that the in this case, activity and the exceptionEvent are referencing the same object.
I've updated the assertions to be using activity to be consistent with the existing code anyhow.